video/out/wayland_common.h

```diff
@@ -85,6 +85,23 @@ struct vo_wayland_state {
     int timeout_count;
     int wakeup_pipe[2];
 
+    /* color-management */
+    struct xx_color_manager_v4 *color_manager;
+    struct xx_color_management_surface_v4 *color_surface;
+    struct xx_image_description_v4 *image_description;
+    struct xx_image_description_creator_params_v4 *image_creator_params;
+    bool supports_icc;
+    bool supports_parametric;
+    bool supports_primaries;
+    bool supports_primaries_named;
+    bool supports_tf_named;
+    bool supports_tf_power;
+    bool supports_luminances;
+    bool supports_display_primaries;
+    bool unsupported_colorspace;
```

Comment:
Could we use bitmap for this? Looks little bit bloated for only `supports` defines.

Reply:
Most of these are not actually used or even needed. I could just eliminate them for now.

Reply:
It is fine to keep it. I though to make it more concise, but whatever works is good.

Reply:
I cut out a couple that definitely aren't needed at least.